core/controllers/domain_objects_validator_test.py

```diff
@@ -221,23 +248,29 @@ def test_invalid_object_raises_exception(self) -> None:
             'card_is_checkpoint': False,
             'solicit_answer_details': False
         }
+        # TODO(#13059): Here we use MyPy ignore because after we fully type the
+        # codebase we plan to get rid of the tests that intentionally test wrong
+        # inputs that we can normally catch by typing.
         # The error is representing the keyerror.
-        with self.assertRaisesRegex(Exception, 'content'): # type: ignore[no-untyped-call]
-            domain_objects_validator.validate_state_dict(invalid_state_dict)
+        with self.assertRaisesRegex(Exception, 'content'):
+            domain_objects_validator.validate_state_dict(invalid_state_dict)  # type: ignore[arg-type]
 
 
 class ValidateSuggestionImagesTests(test_utils.GenericTestBase):
     """Tests to validate suggestion images coming from frontend."""
 
+    # TODO(#13059): Here we use MyPy ignore because after we fully type the
+    # codebase we plan to get rid of the tests that intentionally test wrong
+    # inputs that we can normally catch by typing.
     def test_invalid_images_raises_exception(self) -> None:
         files = {'file.svg': None}
-        with self.assertRaisesRegex( # type: ignore[no-untyped-call]
+        with self.assertRaisesRegex(
             Exception, 'No image supplied'
         ):
-            domain_objects_validator.validate_suggestion_images(files)
+            domain_objects_validator.validate_suggestion_images(files)  # type: ignore[arg-type]
 
     def test_valid_images_do_not_raises_exception(self) -> None:
-        files = {'img.png': None, 'test2_svg.svg': None}
+        files = {'img.png': b'', 'test2_svg.svg': b''}
```

Comment:
Why did this change?

Reply:
> Why did this change?

Because `{'img.png': None, 'test2_svg.svg': None}` makes `files` variable of Dict[str, None] type but `domain_objects_validator.validate_suggestion_images` method can only accept Dict[str, bytes] type. So, to make the type correct I did this change. 

However, the test of None case is not removed, please see the test case just above this one `test_invalid_images_raises_exception`.